documentation/source/users/rmg/database/kinetics.rst

```diff
@@ -211,234 +211,181 @@ Each reaction family contains the files:
-
-.. image:: images/kinetics_families/HO2_Elimination_from_PeroxyRadical.png 
-	:scale: 40% 
-
-**Intra_Diels_alder**     
-
-.. image:: images/kinetics_families/Intra_Diels_alder.png 
-	:scale: 40% 
-
-**Intra_Disproportionation**     
-
-.. image:: images/kinetics_families/Intra_Disproportionation.png 
-	:scale: 40% 
-
-**intra_H_migration**     
-
-.. image:: images/kinetics_families/intra_H_migration.png 
-	:scale: 40% 
-
-**intra_NO2_ONO_conversion**     
-
-.. image:: images/kinetics_families/intra_NO2_ONO_conversion.png 
-	:scale: 40% 
-
-**intra_OH_migration**     
-
-.. image:: images/kinetics_families/intra_OH_migration.png 
-	:scale: 40% 
-
-**Intra_R_Add_Endocyclic**     
-
-.. image:: images/kinetics_families/Intra_R_Add_Endocyclic.png 
-	:scale: 40% 
-
-**Intra_R_Add_Exocyclic**     
-
-.. image:: images/kinetics_families/Intra_R_Add_Exocyclic.png 
-	:scale: 40% 
-
-**Intra_R_Add_ExoTetCyclic**     
-
-.. image:: images/kinetics_families/Intra_R_Add_ExoTetCyclic.png 
-	:scale: 40% 
-
-**Intra_RH_Add_Endocyclic**     
-
-.. image:: images/kinetics_families/Intra_RH_Add_Endocyclic.png 
-	:scale: 40% 
-
-**Intra_RH_Add_Exocyclic**     
-
-.. image:: images/kinetics_families/Intra_RH_Add_Exocyclic.png 
-	:scale: 40% 
-
-**intra_substitutionCS_cyclization**     
-
-.. image:: images/kinetics_families/intra_substitutionCS_cyclization.png 
-	:scale: 40% 
-
-**intra_substitutionCS_isomerization**     
-
-.. image:: images/kinetics_families/intra_substitutionCS_isomerization.png 
-	:scale: 40% 
-
-**intra_substitutionS_cyclization**     
-
-.. image:: images/kinetics_families/intra_substitutionS_cyclization.png 
-	:scale: 40% 
-
-**intra_substitutionS_isomerization**     
-
-.. image:: images/kinetics_families/intra_substitutionS_isomerization.png 
-	:scale: 40% 
-
-**ketoenol**     
-
-.. image:: images/kinetics_families/ketoenol.png 
-	:scale: 40% 
-
-**Korcek_step1**     
-
-.. image:: images/kinetics_families/Korcek_step1.png 
-	:scale: 40% 
-
-**Korcek_step2**     
-
-.. image:: images/kinetics_families/Korcek_step2.png 
-	:scale: 40% 
-
-**lone_electron_pair_bond**     
-
-.. image:: images/kinetics_families/lone_electron_pair_bond.png 
-	:scale: 40% 
-
-**Oa_R_Recombination**     
-
-.. image:: images/kinetics_families/Oa_R_Recombination.png 
-	:scale: 40% 
-
-**R_Addition_COm**     
-
-.. image:: images/kinetics_families/R_Addition_COm.png 
-	:scale: 40% 
-
-**R_Addition_CSm**     
-
-.. image:: images/kinetics_families/R_Addition_CSm.png 
-	:scale: 40% 
-
-**R_Addition_MultipleBond**     
-
-.. image:: images/kinetics_families/R_Addition_MultipleBond.png 
-	:scale: 40% 
-
-**R_Recombination**     
-
-.. image:: images/kinetics_families/R_Recombination.png 
-	:scale: 40% 
-
-**Substitution_O**     
-
-.. image:: images/kinetics_families/Substitution_O.png 
-	:scale: 40% 
-
-**SubstitutionS**     
-
-.. image:: images/kinetics_families/SubstitutionS.png 
-	:scale: 40% 
-
-
+There are currently 58 reaction families in RMG:
```

Comment:
This table of images is giving me:  
System Message: ERROR/3 (/Users/mattjohnson/RMGCODE/RMG-Py/documentation/source/users/rmg/database/kinetics.rst, line 329)

Error in “table” directive: invalid option block.

Reply:
What version of docutils and sphinx do you have?